Cargo.lock

```diff
@@ -1578,9 +1578,9 @@ dependencies = [
 
 [[package]]
 name = "celestia-tendermint"
-version = "0.32.2"
```

Comment:
This is going down for some reason - please fix?

Reply:
it was actually 0.32.1 in Cargo.toml - bumped the version there

Reply:
This should not be necessary. Are you basing this patch on recent `main`? The entry in `Cargo.toml` indicates a minimum version. The entry in `Cargo.lock` indicates the version we actually use.

I have just forked off recent main and updated the penumbra entries - `celestia-tendermint` and `celestia-tendermint-proto` are not affected by the penumbra change.

If you take the current main, do the penumbra changes there, and then regenerate `Cargo.lock` you should get a clean patch.

Reply:
I rebased on top of main, reverted the last commit (for tendermint), and then restored && regenerated Cargo.lock. Now it does not contain tendermint changes.